Cargo.toml

```diff
@@ -66,7 +66,7 @@ gix = { version = "0.66.0", default-features = false, features = [
 ] }
 gix-filter = "0.13.0"
 glob = "0.3.1"
-hex = "0.4.3"
+faster-hex = { version = "0.9.0", default-features = false, features = ["std"]}
```

Comment:
nit: apparently, 0.10.0 is released yesterday

and the dependency list is sorted alphabetically.

Reply:
I didnt wanna depend on that due to us already depending on 0.9.0 transitively and those dependencies aren't bumpable